### DISCUSSION TITLE
KAMP-for-K1-Series shows Repo is corrupt

### DISCUSSION THREAD

**Discussion by steveharman** [Guilouz/Creality-Helper-Script-Wiki#460] [Issues]
Prior to the latest script which incorporates the KAMP toggle, I had followed instructions here and manually implemented the toggle option.  Since that point `Fluidd -> System -> Software update` showed "Repo is corrupt":

![Notification_Center](https://github.com/Guilouz/Creality-K1-Series/assets/12557929/4d026499-6132-4083-a425-ffb8b1c26d0d)

I am now on the latest script but the error persists.  I've tried removing and reinstalling KAMP support but that didn't help.

I tried using the `Recover` option in Software Update, which results in this error:

```
Git Repo KAMP-for-K1-Series: Attempting Repo Recovery...
Git Repo KAMP-for-K1-Series: Resetting Git Repo...
fatal: could not read Username for 'https://github.com': No such device or address
Attempting to repair loose objects...
Attempt to repair loose objects failed, hard recovery is required
Error Recovering KAMP-for-K1-Series
Git Repo KAMP-for-K1-Series: Attempting Repo Recovery...
Git Repo KAMP-for-K1-Series: Starting Clone Recovery...
Cloning into '/usr/data/.KAMP-for-K1-Series_repo_backup'...
fatal: could not read Username for 'https://github.com': No such device or address
Git Repo KAMP-for-K1-Series: Git Clone Failed
Error Recovering KAMP-for-K1-Series
```
Now that the KAMP toggle is supported within the script I'd just like to forget about the manual change and have a clean repo like all the other components.  How can I achieve this please?

**Comment by alkattaf** [comment]
Re install  root and everything's again 
There are alot of change with the new update

**Reply by steveharman** [reply]
ok, thanks.  And start with a full factory reset?  As per: https://guilouz.github.io/Creality-K1-Series/firmwares/reset-factory-settings/

**Reply by alkattaf** [reply]
Yes , full factory reset , then root again , you can check the change log , more future added

**Reply by steveharman** [reply]
Great, many thanks.

**Comment by leseaw** [comment]
https://github.com/Guilouz/KAMP-for-K1-Series

This is because the site has moved.
Click on the small square with the arrow and you will see the old url in it
